tox.ini

```
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 100
```

```suggestion
max-line-length = 121
```
I like longer lines, but 100 is fine too.

---

Maybe something like 119, so it's exactly 40 chars above.

---

111?